### PR TITLE
Java 19 release will be called 0.37.0

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
https://github.com/eclipse-openj9/openj9-docs/issues/1042

Updated the supported environments topic and the What's new topic along with the .yml file

 Signed-off-by: Sreekala Gopakumar <sreekala.gopakumar@ibm.com>

### CODE REVIEW

**Comment by pshipton** [comment]
@AdamBrousseau @doveye the doc staging https://openj9-jenkins.osuosl.org/job/PullRequest-Doc-test_on_staging_site/248/ failed due to the following. Is this related to the recent changes to update containers and dependencies? I'll try another build just in case.
```
09:02:06  + git status
09:02:06  fatal: unable to create threaded lstat: Operation not permitted
```

Jenkins doc stage

**Comment by AdamBrousseau** [comment]
Unrelated to recent changes. Seems to happen on Ubuntu hosts.

**Comment by AdamBrousseau** [comment]
Jenkins doc stage

File: docs/version0.36.md [hunk 1]
```diff
@@ -38,9 +38,11 @@ The following new features and notable changes since version 0.35.0 are included
 
 ### Binaries and supported environments
 
-Eclipse OpenJ9&trade; release 0.36.0 supports OpenJDK 8, 11, and 17.
+Eclipse OpenJ9&trade; release 0.36.0 supports OpenJDK 8 and 17.
 
-Support for running OpenJDK 8 and OpenJDK 11 on CentOS 6.10 is deprecated and might be removed in a future release. OpenJ9 will not be tested with OpenJDK 11 on CentOS 6.10 after 0.36.0 release.
+Release 0.36.1 supports OpenJDK 11.
+
+Support for running OpenJDK 8 and OpenJDK 11 on CentOS 6.10 is deprecated and might be removed in a future release. OpenJ9 will not be tested with OpenJDK 11 on CentOS 6.10 after 0.36.1 release.
```
Comment (pshipton) [inline docs/version0.36.md:45]: after `the` 0.36.1 release

File: docs/openj9_support.md [hunk 2]
```diff
@@ -46,10 +46,11 @@ columns will be removed over time.
...
-| 0.38.0         | Apr 2023 (1)  | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>     |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>          |
+| 0.36.0         | Jan 2023  | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     |      |            |
+| 0.36.1         | Mar 2023  | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>     |     |            |
```
Comment (pshipton) [inline docs/openj9_support.md:50]: The release date for 0.36.1 is Feb now, similarly for 0.36.0

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Review by pshipton — APPROVED** [review]

**Pull request merged**